### PR TITLE
chore: created new ci for ECR

### PR DESCRIPTION
**Pull request by lyonely**
notion story: https://www.notion.so/aipf-locus/ECR-Github-Actions-sg-tileserver-gl-fadea6c27ca44b35b4b41c15413b2f7f?pvs=4

add new GHA workflow to upload docker images to ECR

### CODE REVIEW

**Review by tyng94 — APPROVED** [review]
see the images on ECR, thanks

File: .github/workflows/ci-ecr.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          --target native \
+          --build-arg TILESERVER_GL_VERSION="${TILESERVER_GL_VERSION}"
+    - name: Build Docker image (with openresty)
+      run: |-
+        docker build . -t "${IMAGE_NAME}:${TAG_NAME}${OPENRESTY_SUFFIX}" \
+          --build-arg TILESERVER_GL_VERSION="${TILESERVER_GL_VERSION}"
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        audience: sts.amazonaws.com
```
Comment (Tingweiftw) [inline .github/workflows/ci-ecr.yml:77]: remove

File: .github/workflows/ci-ecr.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+on:
+  push:
+    branches:
+    - main
+    - v*
+  pull_request:
+    branches:
+    - main
+    - v*
```
Comment (Tingweiftw) [inline .github/workflows/ci-ecr.yml:11]: add release trigger

**Comment by Tingweiftw** [comment]
removed sts here

**Pull request merged**